jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Session.java

```diff
@@ -82,7 +83,7 @@
 import org.slf4j.LoggerFactory;
 
 @ManagedObject
-public abstract class HTTP2Session extends ContainerLifeCycle implements Session, Parser.Listener
+public abstract class HTTP2Session extends AbstractLifeCycle implements Session, Parser.Listener, Dumpable
```

Comment:
Is this a change we can make in 12.0?    Could a user be expect session to be a container and add beans to it?

Reply:
Applications should not access this implementation class.
It is not internal because it is used by both client and server, but it is "internal" for applications.